.config.json

```diff
@@ -1,9 +1,9 @@
 {
-	"host": "http://mattermost.example.com:8065",
-	"port": 8505,
-	"token": "9jrxak1ykxrmnaed9cps9i4cim",
-	"user": {
-		"id": "bot",
-		"password": "botbot"
-	}
+  "host": "http://mattermost.example.com:8065",
+  "listen": "localhost:8505",
```

Comment:
It could be either but I left it this way because it's more secure and I imagine most will have Mattermost and this project running on the same server. The problem with `:8505` is it will listen on all interfaces which means you could be listening on an interface which is facing the internet or another network segment unintentionally. There's too many possible scenarios where that could be a problem so it's best to air on the side of caution in my opinion.